src/main/kotlin/uk/gov/dluhc/printapi/rds/repository/CertificateRepository.kt

```diff
@@ -2,8 +2,16 @@ package uk.gov.dluhc.printapi.rds.repository
 
 import org.springframework.data.jpa.repository.JpaRepository
 import org.springframework.stereotype.Repository
+import uk.gov.dluhc.printapi.database.entity.Status
 import uk.gov.dluhc.printapi.rds.entity.Certificate
 import java.util.UUID
 
 @Repository
-interface CertificateRepository : JpaRepository<Certificate, UUID>
+interface CertificateRepository : JpaRepository<Certificate, UUID> {
+
+    fun getByPrintRequestsRequestIdIs(requestId: String): Certificate
+
+    fun findByStatusAndPrintRequestsBatchIdIs(certificateStatus: Status, batchId: String): List<Certificate>
+
+    fun findByStatusIs(certificateStatus: Status): List<Certificate>
```

Comment:
Minor naming .... method names suffixed with `Is` ? Do we do that elsewhere? 🤔 
Would `findByStatus` etc be more in keeping with what we do elsewhere?

Reply:
I copied from `VoterCardApplicationRepository` but agree it reads better without the `Is` so have removed.